### DISCUSSION TITLE
could not open /dev/vmmon error when run ova

### DISCUSSION THREAD

**Discussion by fyqphd** [Seagate/cortx#426] [General] [answered]
followed this guidance https://github.com/Seagate/cortx/blob/main/doc/CORTX_on_Open_Virtual_Appliance.rst

downloaded vmware fusion and cortx-va-1.0.1.zip
imported cortx-va-1.0.1 to vmware fusion

![Screenshot 2020-10-23 at 10 29 06 PM](https://user-images.githubusercontent.com/49628448/97017579-adc07300-1580-11eb-8860-a41c1bcc54e3.png)

when run the vm, the following error shows

![Screenshot 2020-10-23 at 10 30 06 PM](https://user-images.githubusercontent.com/49628448/97017622-bb75f880-1580-11eb-9cb4-fb6c8629db81.png)

my mac info

Model Name:	MacBook Pro
  Model Identifier:	MacBookPro16,1
  Processor Name:	8-Core Intel Core i9
  Processor Speed:	2.4 GHz
  Number of Processors:	1
  Total Number of Cores:	8
  L2 Cache (per Core):	256 KB
  L3 Cache:	16 MB
  Hyper-Threading Technology:	Enabled
  Memory:	32 GB

Macintosh HD:

  Free:	1.94 TB (1,939,279,597,568 bytes)
  Capacity:	2 TB (2,000,796,545,024 bytes)
  Physical Drive:
  Device Name:	APPLE SSD AP2048N
  Media Name:	AppleAPFSMedia
  Medium Type:	SSD
  Protocol:	PCI-Express
  Internal:	Yes
  Partition Map Type:	Unknown
  SMART Status:	Verified

[a comment by an automated account is not shown]

**Comment by justinzw** [comment]
Hi @fyqphd  Thank you for reporting this and also trying out the instructions.

We've had some difficulties running the OVA on older versions of VMware Fusion.

Which version of VMware Fusion are you using? 
If you don't have the latest version  - 12.  

Could you try downloading the latest version and see whether it works?
https://www.vmware.com/products/fusion/fusion-evaluation.html

**Comment by fyqphd** [comment]
@justinzw   I beleive I did download the version 12

<img width="1053" alt="Screenshot 2020-10-24 at 8 17 21 AM" src="https://user-images.githubusercontent.com/49628448/97063283-67473480-15d1-11eb-8494-17a3b78e83ed.png">

**Comment by justinzw** [comment, marked as answer]
Thanks @fyqphd for checking. 

Could you try these instructions and see whether they work? 

https://kb.vmware.com/s/article/80467

**Reply by justinzw** [reply]
This worked for @fyqphd. We'll look into adding it in the documentation.

@VenkyOS @johnbent

**Reply by VenkyOS** [reply]
@justinzw - I have added a link to this document in the Hypervisors document.
